ratpack/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] [%X{userName}] [%X{TraceId}] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
ratpack-zipkin doesn't use our scope decorators at the moment. Instead, it explicitly adds this TraceId property. https://github.com/openzipkin-contrib/brave-ratpack/blob/master/src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java#L55